tests/cp2/test_cp2_cbez.s

```diff
@@ -34,12 +34,14 @@
 
 #
 # Test CBEZ (capability branch if NULL).
+# Note: This only checks the address (a valid cap with address 0 is "NULL")
 #
 
 BEGIN_TEST
 		dli	$a0, 0
 		dli	$a2, 0
-		cgetnull $c22
+		cgetdefault	$c22
+		csetaddr	$c22, $c22, $zero	# Ensure zero address
```

Comment:
Technically unnecessary for the tests to pass but I do like that these diffs mean we test that it's looking at _only_ the address